src/anim/controller/anim-blend-tree-1d.js

```diff
@@ -35,6 +40,11 @@ class AnimBlendTree1D extends AnimBlendTree {
                 c2.weight = 0.0;
             }
         }
+        if (this._syncDurations) {
+            for (i = 0; i < this._children.length; i++) {
+                this._children[i].weightedSpeed = meanDuration;
```

Comment:
it's probably a naming again, but `weightedSpeed = meanDuration` seems to assign duration to speed? Also, 'mean' is an average, but in this case it seems to be a sum of durations?

Reply:
This confusion must come from the fact that part of the calculations for the mean duration are occurring within the property getters/setters for weightedSpeed and weightedDuration. I can move those calculations out of there.